libs/core/langchain_core/prompts/chat.py

```diff
@@ -526,6 +529,34 @@ def format(self, **kwargs: Any) -> BaseMessage:
                 content=content, additional_kwargs=self.additional_kwargs
             )
 
+    async def aformat(self, **kwargs: Any) -> BaseMessage:
```

Comment:
is there a unit test we can hook into for this function to prevent the implementations from drifting?

Reply:
Yes, this code path is invoked by the async tests in `tests/unit_tests/prompts/test_few_shot.py`

Reply:
But I'll also add async tests in `test_chat.py` for the multipart text.

Reply:
done.
@eyurtsev can you take a look again ?

Reply:
Thank you!